app/controllers/tasks_controller.rb

```diff
@@ -10,14 +10,26 @@ def create
     redirect_to list_path(@list)
   end
 
+  def show
+    @task = Task.find(params[:id])
+    @list = @task.list_id
```

Comment:
It seems a bit odd to assign a list id to an `@list` variable. I would recommend changing the variable to `@list_id` or doing `@task.list` so you get the whole list object back